### ISSUE TITLE
NullPointerException without warmup

### ISSUE THREAD

**Issue by simeonackermann** [AKSW/IGUANA#18]
Hi,

without the warmup"tag in my config, I get the following error on trying to run a benchmark:

```
Exception in thread "main" java.lang.NullPointerException
	at java.io.File.<init>(File.java:277)
	at de.uni_leipzig.iguana.utils.FileHandler.getQueriesInFile(FileHandler.java:317)
	at de.uni_leipzig.iguana.benchmark.processor.WarmupProcessor.warmup(WarmupProcessor.java:102)
	at de.uni_leipzig.iguana.benchmark.processor.TestcaseProcessor.testTestcase(TestcaseProcessor.java:104)
	at de.uni_leipzig.iguana.benchmark.Benchmark.start(Benchmark.java:105)
	at de.uni_leipzig.iguana.benchmark.Benchmark.execute(Benchmark.java:49)
	at de.uni_leipzig.iguana.benchmark.Main.main(Main.java:40)
```

If I include the tag it runs fine, even if the warmup-file is empty or does not exist.

**Comment by TortugaAttack** [comment]
Oh okay, i'll fix it tomorrow. 
If it's empty it will be ignored, but i'll write a loig message for that case.
Right now you simply can set the warmup time to 0 :)

Thanks alot

**Comment by TortugaAttack** [comment]
Fixed with the 0.3.0.2 version 
Be aware that the connection specs in the config file has changed so the endpoint is not anymore 
"localhost:8080/..." but "http://localhost:8080/..." to enable https connections ;)

**Comment by simeonackermann** [comment]
Cool, very fast!
And thanks for the hint.
